### ISSUE TITLE
UpdatedDateUTC is not UTC but local time instead

### ISSUE THREAD

**Issue by ngvtien** [XeroAPI/Xero-NetStandard#149]
**SDK you're using (please complete the following information):**
 - Current

**Describe the bug**
After posting an invoice with the current API SDK the property "UpdatedDateUTC" in the Invoice model didn't come back as UTC like what it suppose to but instead, it came back as local time

**Comment by SidneyAllen** [comment]
Duplicate of #121

**Comment by jenksguo** [comment]
See #121 , issue will be addressed in the next release.

**Comment by jenksguo** [comment]
Hi @ngvtien thanks for reporting the issue. 

It has taken some time but we managed to eventually fixed this issue. 

The UpdatedDateUTC now is converted to a DateTime object with DateTime.Kind of Utc: 

![image](https://user-images.githubusercontent.com/41350731/80855015-7a132900-8c80-11ea-84cd-6a9ad01b6ce4.png)

This is now released in [v2.0.1](https://github.com/XeroAPI/Xero-NetStandard/releases/tag/v2.0.1) of the SDK. 

Please note that there is a breaking change between your version and the current SDK. I have changed the namespace of models so you will need to update your inclusion, 

from: 
```csharp
using Xero.NetStandard.OAuth2.Model;
```
to: 
```csharp
using Xero.NetStandard.OAuth2.Model.Accounting;
```
